### PR TITLE
test: remove duplicated chromatic in parameters

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- remove duplicated part of the code for disabling chromatic in each story (parameters in `component.story` section)
- disable `chromatic` for Loader, AnchorNavigation, Duelling Picklist, and Tippy because we can't make a `visual` regression for components with opening via the button or rendering in portal.

### Current behaviour

We need to:
- remove duplicated part of the code for disabling chromatic in each story
- disable also Loader, AnchorNavigation, Duelling Picklist and Tippy stories from chromatic

### Checklist

- [x] Commits follow our style guide
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Carbon implementation and Design System documentation are congruent

### Testing instructions

Check are all stories covered by chromatic

### CODE REVIEW

File: src/components/anchor-navigation/anchor-navigation.stories.js [hunk 1]
```diff
@@ -222,7 +222,7 @@ InFullScreenDialog.story = {
   name: "in full screen dialog",
   parameters: {
     chromatic: {
-      disable: false,
+      disable: true,
```
Comment (edleeks87) [inline src/components/anchor-navigation/anchor-navigation.stories.js:225]:
```suggestion
      disabled: true,
```

File: src/components/accordion/accordion.stories.js [hunk 2]
```diff
@@ -70,7 +67,7 @@ Grouped.story = {
   name: "grouped",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/accordion/accordion.stories.js:70]: should this not be `disable`?

File: src/components/accordion/accordion.stories.js [hunk 3]
```diff
@@ -61,7 +58,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/accordion/accordion.stories.js:61]: should this not be disable?

File: src/__experimental__/components/search/search.stories.js [hunk 4]
```diff
@@ -62,7 +59,8 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/__experimental__/components/search/search.stories.js:62]: should this not be disable?

File: src/components/app-wrapper/app-wrapper.stories.js [hunk 5]
```diff
@@ -29,7 +26,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/app-wrapper/app-wrapper.stories.js:29]: should this not be disable?

File: src/components/content/content.stories.js [hunk 6]
```diff
@@ -43,7 +40,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/content/content.stories.js:43]: should this not be disable?

File: src/components/definition-list/definition-list.stories.js [hunk 7]
```diff
@@ -13,7 +13,7 @@ export default {
     },
     knobs: { escapeHTML: false },
     chromatic: {
-      disabled: true,
+      disabled: false,
```
Comment (edleeks87) [inline src/components/definition-list/definition-list.stories.js:16]: disable?

File: src/components/detail/detail.stories.js [hunk 8]
```diff
@@ -33,7 +30,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/detail/detail.stories.js:33]: disable?

File: src/components/heading/heading.stories.js [hunk 9]
```diff
@@ -45,7 +42,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/heading/heading.stories.js:45]: should this not be disable?

File: src/components/i18n/i18n.stories.js [hunk 10]
```diff
@@ -34,7 +31,7 @@ Default.story = {
   name: "default",
   parameters: {
     chromatic: {
-      disable: true,
+      disabled: true,
```
Comment (edleeks87) [inline src/components/i18n/i18n.stories.js:34]: should this not be disable?

**Review by edleeks87 — COMMENTED** [review]
I think `disabled` needs to be `disable` according to https://www.chromatic.com/docs/ignoring-elements

**Review by edleeks87 — APPROVED** [review]

**Review by mkrds — APPROVED** [review]

**Review by KatarzynaQA — APPROVED** [review]

**Pull request merged**